app/helpers/application_helper.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def title(page_title)
+    content_for(:title) { page_title }
```

Comment:
Could we set a default page title? I'm trying to think of a way to avoid the separator being included every time we set the title.